### PR TITLE
JIRAID: items are added in the list

### PR DESCRIPTION
**Pull request by ghost**
Can you please review this

### CODE REVIEW

File: addcart.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Item list 
+
+WhiteBoard
+Notebook
+Pencils
```
Comment (yogskarma) [inline addcart.html]: ADD NUMBERING

**Comment by ghost** [comment]
Can you plese review this.

File: addcart.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>AddCart</title>
+</head>
+<body>
+<li>1.White board</li>
```
Comment (yogskarma) [inline addcart.html]: Will not work , will show number twice

**Review by yogskarma — COMMENTED** [review]
please fix the issue

**Comment by ghost** [comment]
Review this change

**Review by yogskarma — APPROVED** [review]
Looks good

**Pull request merged**